projects/market/src/app/skills/install-button/install-button.component.scss

```diff
@@ -42,8 +60,8 @@ fa-icon {
 
 #installing-button {
     @include install-status;
-    background-color: mat-color($mycroft-accent, A100);;
-    color: mat-color($mycroft-accent);
+    background-color: mat.get-color-from-palette(theme.$mycroft-accent, A100);;
+    color: mat.get-color-from-palette(theme.$mycroft-accent);
```

Comment:
do these always need a second arg to select the accent shade or will a default be used?
I think I saw this a few times, so if it's an issue would need to search for other instances.

Reply:
good catch.  since the install button is hidden, I didn't test this.  fixed.